GettingStarted/ibmdbpy_GettingStarted_2-geo_basics.ipynb

```diff
@@ -0,0 +1,1621 @@
+       "2      047  01035620      Logan  38047      Logan County   06      H1  G4020   \n",
+       "3      067  00198148   La Plata  08067   La Plata County   06      H1  G4020   \n",
+       "4      121  00069178   Randolph  05121   Randolph County   06      H1  G4020   \n",
+       "\n",
+       "  CSAFP CBSAFP METDIVFP FUNCSTAT         ALAND      AWATER     INTPTLAT  \\\n",
+       "0        46900                 A  2.514474e+09  18257915.0  +34.0849199   \n",
+       "1   288  26420                 A  1.674401e+09  25610780.0  +29.8919013   \n",
+       "2                              A  2.571371e+09  47715597.0  +46.4692780   \n",
+       "3        20420                 A  4.382664e+09  19545452.0  +37.2873673   \n",
+       "4                              A  1.689166e+09   9968439.0  +36.3412985   \n",
+       "\n",
+       "       INTPTLON  \n",
+       "0  -099.2424397  \n",
+       "1  -096.2701696  \n",
+       "2  -099.5045846  \n",
+       "3  -107.8397178  \n",
+       "4  -091.0284409  "
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "idageodf.head()\n",
+    "# what about this warning ?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* Method 2: convert an IdaDataFrame with eligible geometry column into an IdaGeoDataFrame\n",
+    "\n",
+    "You can create an IdaGeoDataFrame by converting an IdaDataFrame which you gave an eligible geometry column. Note that an IdaGeoDataFrame inherits all the properties of an IdaDataFrame, so you could also simply open it as an IdaDataFrame if you do not need to manipulate its geospatial information, and convert it back any time into an IdaGeoDataFrame. For conversion, use the `IdaGeoDataFrame.from_IdaDataFrame` function. Note that in any case, an IdaGeoDataFrame is characterized by the presence of a column with geometry type: its data type belongs to the `ST_Geometry` family (like in Db2 : `db2gse.ST_Geometry`). "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# here we open the Db2 table SAMPLES.GEO_COUNTY as IdaDataFrame\n",
+    "idadf = IdaDataFrame(idadb, 'SAMPLES.GEO_COUNTY', indexer='OBJECTID')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# now we convert this IdaDataFrame into an IdaGeoDataFrame\n",
+    "# we set the 'SHAPE' column as geometry\n",
+    "geo_idadf = IdaGeoDataFrame.from_IdaDataFrame(idadf, geometry = \"SHAPE\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "__Understand the geometry column__\n",
+    "\n",
+    "Have a look at the data types : you see that the `SHAPE` column is of type `ST_MULTIPOLYGON`. What does that mean? It is a geospatial data type inherited fron Db2gse, Db2 geospatial extension."
```

Comment:
```suggestion
    "Have a look at the data types : you see that the `SHAPE` column is of type `ST_MULTIPOLYGON`. What does that mean? It is a geospatial data type inherited from Db2gse, Db2 geospatial extension."
```

Also, db2gse is officially the [Db2 Spatial Extender](https://www.ibm.com/support/knowledgecenter/SSCJDQ/com.ibm.swg.im.dashdb.spatial.doc/doc/csbp1001.html) (don't ask me about the `g` :-D ).

Reply:
ok I have changed it to the official name... well apparently those who chose the official name where not the actual developers of db2gse ;)